pkg/machine/qemu/machine_windows.go

```diff
@@ -0,0 +1,27 @@
+package qemu
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/containers/podman/v4/pkg/machine/wsl"
```

Comment:
Could you move the GetProcessState function to the pkg/machine package (e.g. machine_windows.go) and import pkg/machine instead?

This is where we have shared code already, and the change addresses two benefits:
- Cycles are less likely
- The WSL and Qemu backends are not tied together, and build tags can potentially exclude one or the other.

Reply:
Moved the utility to shared function in machine package.